bin/cmd.js

```diff
@@ -110,6 +112,31 @@ gcr.load(parsed, function(err) {
 
   if (!gcr.config.get('token')) {
     questions.push(tokenQuestion)
+    if (!gcr.config.get('registerToken')) {
+	questions.push(tokenQuestion)
```

Comment:
If `token` is false and `registerToken` is true, this will push `tokenQuestion` twice.